localization/replace/english/wc_title_replacements_l_english.yml

```diff
@@ -229,4 +229,10 @@
  b_ramu:0 "Ra'mu"
  b_ronda:0 "Ron'da"
  b_larne:0 "Lar'ne"
- b_carrion:0 "Carrion"
\ No newline at end of file
+ b_carrion:0 "Carrion"
+ 
+ d_strand_of_ancients:0 "Alduran"
+ d_strand_of_ancients_adj:0 "Aldurani"
+
+ d_stormsong:0 "Briarback"
+ d_stormsong_adj:0 "Briarback"
```

Comment:
So why is Stormsong called Briarback when not controlled by humans?
Shouldn't it be called Briarback only when controlled by quillboars?

Reply:
fixed